pages/common/xargs.md

```diff
@@ -10,6 +10,10 @@
 
 `{{arguments_null_terminated}} | xargs -0 {{command}}`
 
-- example: list unneeded packages with deborphan and remove them with apt-get
+- example: delete all files that start with 'M'
 
-`sudo deborphan | xargs sudo apt-get remove`
+`ls M* | xargs rm`
+
+- creating variables: copy all files that start with 'M' to the Desktop
```

Comment:
I would say it needs more explaining. It is unclear.

This is from man. It does not make it clear too.

     -J replstr
             If this option is specified, xargs will use the data read from
             standard input to replace the first occurrence of replstr instead
             of appending that data after all other arguments.  This option
             will not affect how many arguments will be read from input (-n),
             or the size of the command(s) xargs will generate (-s).  The
             option just moves where those arguments will be placed in the
             command(s) that are executed.  The replstr must show up as a dis-
             tinct argument to xargs.  It will not be recognized if, for
             instance, it is in the middle of a quoted string.  Furthermore,
             only the first occurrence of the replstr will be replaced.  For
             example, the following command will copy the list of files and
             directories which start with an uppercase letter in the current
             directory to destdir:

                   /bin/ls -1d [A-Z]* | xargs -J % cp -rp % destdir